include/fmt/chrono.h

```diff
@@ -18,6 +18,33 @@
 
 FMT_BEGIN_NAMESPACE
 
+template <typename Period> FMT_CONSTEXPR const char* get_units() {
```

Comment:
This and specializations should probably be internal.

Reply:
Right, done